src/service/data_products/heatmap_common_models.py

```diff
@@ -72,10 +90,12 @@ class Cell(BaseModel):
         example="8",
         description="The ID of the column in which this cell is located."
     )
-    val: float = Field(
+    val: float | bool = Field(
```

Comment:
we probably also need `int` type here. Otherwise, the column type might be `int` and all cell values are automatically converted to float int.

Reply:
done

Reply:
👍